core/appconfig/debug_container.dot

```diff
@@ -0,0 +1,3 @@
+digraph "" {
```

Comment:
what's dis?

Reply:
hmm seems to be something that gets added when i run make test internally, i removed it